app/dts/bindings/animations/zmk,animation-control.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020, The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: |
+  Higher-order animation which allows for controlling animation drivers
+  placed underneath it by turning them on and off, cycling though them,
+  or changing the brightness.
+
+compatible: "zmk,animation-control"
+
+properties:
+  label:
+    type: string
+    required: true
+
+  animations:
+    type: phandles
+    required: true
+    description: |
+      Animations to be combined.
+
+  brightness-steps:
+    type: int
+    required: false
```

Comment:
I believe this is true as of Zephyr 3.2 (it definitely is for 3.4)- `required: false` should not be required, and can be removed.